calico/about/about-calico.mdx

```diff
@@ -0,0 +1,68 @@
+---
+description: A brief description of Calico, deployment options, and features.
+---
+
+# About Calico 
+
+## What is {{prodname}}?
+
+{{prodname}} is a free, self-managed networking and security solution for containers, virtual machines, and native host-based workloads. {{prodname}} provides networking, network policy, and IP address management for cloud-native applications. 
+
+## Deployment options
+
+{{prodname}} is compromised of two decoupled components that offer the widest deployment options across platforms:
+
+- **{{prodname}} CNI** to configure networking resources
+- **{{prodname}} network policy** to secure workload and host endpoints
+
+For example, if you use Kubernetes for AWS (self-managed) or EKS (public cloud), you can use either of these options:
+
+- {{prodname}} CNI and {{prodname}} network policy
+- AWS VPC CNI with {{prodname}} network policy 
+
+To understand the {{prodname}} deployment options, check out these popular platforms:
+
+- [AKS](../getting-started/kubernetes/managed-public-cloud/aks.mdx)
+- [Azure](../getting-started/kubernetes/self-managed-public-cloud/azure.mdx)
+- [Digital Ocean](../getting-started/kubernetes/self-managed-public-cloud/do.mdx)
+- [EKS](../getting-started/kubernetes/managed-public-cloud/eks.mdx)
+- [Flannel/Canal](../getting-started/kubernetes/flannel/install-for-flannel.mdx)
+- [GCE](../getting-started/kubernetes/self-managed-public-cloud/gce.mdx)
+- [GKE](../getting-started/kubernetes/managed-public-cloud/gke.mdx)
+- [IKS](../getting-started/kubernetes/managed-public-cloud/iks.mdx)
+- [K3s](../getting-started/kubernetes/k3s/index.mdx)
+- [kOps on AWS](../getting-started/kubernetes/self-managed-public-cloud/aws.mdx)
+- [Kubernetes-kubeadm](../getting-started/kubernetes/self-managed-onprem/onpremises.mdx)
+- [MicroK8s](../getting-started/kubernetes/microk8s.mdx)
+- [Minikube](../getting-started/kubernetes/minikube.mdx)
+- [OpenShift](../getting-started/kubernetes/openshift/index.mdx)
+- [OpenStack](../getting-started/openstack/index.mdx)
+- [RKE](../getting-started/kubernetes/rancher.mdx)
+- [Calico for Windows](../getting-started/kubernetes/windows-calico/index.mdx)
+
+## Installation options
+
+You can install {{prodname}} using Kubernetes operators, manifests (YAMLs), or Helm charts (Helm 3). 
```

Comment:
"You can install Calico using a Kubernetes operator, manifests, or a Helm chart"

^ I'd suggest wording like this. We only use a single operator / helm chart. I also think helm 3 is assumed at this point, since helmv2 was deprecated quite some time ago.

Reply:
Should I delete manifest as well? Is it considered legacy?

Reply:
I think it's still worth mentioning manifests - they are still prevalent in OSS installs.

Reply:
Fixed.